src/main/webapp/resources/js/apis/pipelines/pipelines.js

```diff
@@ -27,10 +28,70 @@ export async function getIridaWorkflowDescription(workflowUUID) {
  * Get a listing of all Pipelines in IRIDA.
  * @returns {Promise<AxiosResponse<any> | never>}
  */
-export const fetchIridaAnalysisWorkflows = async function() {
-  var ajaxUrl = URL;
-  if (window.PAGE.automatedProject !== null) {
-    ajaxUrl = `${ajaxUrl}?automatedProject=${window.PAGE.automatedProject}`;
-  }
-  return axios.get(ajaxUrl).then(response => response.data);
+export const fetchIridaAnalysisWorkflows = async function () {
+  return axios.get(AJAX_URL).then((response) => response.data);
+};
+
+export const fetchAutomatedIridaAnalysisWorkflows = async function () {
+  return axios.get(`${AJAX_URL}/automated`).then((response) => response.data);
 };
+
```

Comment:
Also, I notice in this file that there are places we throw a `new Error` and places where we use `Promise.reject`. We should probably stick to `Promise.reject`. Thoughts?

Reply:
I like `Promise.reject` updated in b4e46b6bfc9bc6b6957157ee584fefbc6adca26e